angular/.angular-cli.json

```diff
@@ -34,7 +34,8 @@
         "prod": "environments/environment.prod.ts",
         "prodcompose": "environments/environment.prodcompose.ts",
         "node": "environments/environment.node.ts",
-        "net": "environments/environment.net.ts"
+        "net": "environments/environment.net.ts",
+        "openshift": "environments/environment.openshift.ts"
```

Comment:
I think that we should add also the same changes to the default development environment so we only work with the "standard" angular environments default(dev) and prod

of course projects can create their own environments but we leave that option for more exotic ones, not for the common case

also we should include a comment on the file explaining the logic behind the "extrange" code because it is not that intuitive